sos/report/plugins/opencontrail.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2021 Mirntis, Inc., Oleksii Molchanov <omolchanov@mirantis.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+
+
+class OpenContrail(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
```

Comment:
Recently we've started to use `IndependentPlugin` more for plugins which can do the exact same collections across our supported distros. This also means new policies can pick up more plugins right away without having to insert a new tagging class into each plugin.

Reply:
Fixed to follow what looks to be new most up to date best practice.